### ISSUE TITLE
UI and accessibility 

### ISSUE THREAD

**Issue by qtop1** [sitespeedio/sitespeed.io#4164]
### Feature/improvement

Hi, 

I ran the website through the following accessibility checkers: EXPERTE, Audit and AccessiBe. Overall all the checkers returned a good accessibility score rate, but the most common issue shown in the all reports was the lack of high contrast between the foreground and background colours. I would like to resolve this issue, to help users with vision impairments. Specifically  fixing the tab bar colour ratio against the background.

**Comment by soulgalore** [comment]
Hi @qtop1 ok, sounds good. For the tab in the header one way could be to use the clicked/chosen version of the color as default and then do a darker version of that version when clicking/chosen?

**Comment by qtop1** [comment]
Hi, yh that's a good idea. I may have to go darker than the default color of the clicked option as it almost passes the color contrast analyser test.
<img width="480" alt="Screenshot 2024-06-02 at 17 21 27" src="https://github.com/sitespeedio/sitespeed.io/assets/132226088/8ae0a886-3b48-493e-8cea-69da7c6ac1ac">
